pisa-proxy/protocol/mysql/src/client/conn.rs

```diff
@@ -192,6 +192,28 @@ impl ClientConn {
         res
     }
 
+    pub async fn send_ping(&mut self) -> Result<bool, ProtocolError> {
+        let framed = self.framed.take().unwrap();
+        let mut common_codec = framed.into_common();
+
+        common_codec.send((COM_PING, &[])).await?;
+        let res = match common_codec.next().await {
+            Some(Ok(data)) => {
+                if data.0[4] == OK_HEADER {
```

Comment:
Make it a general function if it is a common need.

Reply:
fix with adding `is_ok_header() `method